### PR TITLE
Log to cloudwatch

### PR DESCRIPTION
**Pull request by SteveHNH**
With these changes we can log to cloudwatch. Basically replaced Zap with Logrus and updated all the log messages

### CODE REVIEW

File: logger/logger.go [hunk 1]
```diff
@@ -2,56 +2,67 @@ package logger
...
+	}
 
-		defer logger.Sync()
-		Log = logger
+	formatter := &logrus.JSONFormatter{
+		FieldMap: logrus.FieldMap{
+			logrus.FieldKeyTime:  "time",
+			logrus.FieldKeyFunc:  "caller",
+			logrus.FieldKeyLevel: "level",
+			logrus.FieldKeyMsg:   "message",
```
Comment (jhjaggars) [inline logger/logger.go]: Should this be `msg`?

File: logger/logger.go [hunk 2]
```diff
@@ -2,56 +2,67 @@ package logger
...
+		},
+	}
+
+	Log.SetFormatter(formatter)
+
+	cred := credentials.NewStaticCredentials(key, secret, "")
+	cfg := aws.NewConfig().WithRegion("us-east-1").WithCredentials(cred)
```
Comment (jhjaggars) [inline logger/logger.go]: Probably should get the region from the environment as well.

File: logger/logger.go (hunk 1 above)
Comment (SteveHNH) [reply logger/logger.go]: It wasn't in the old logger, but we can change it.

File: logger/logger.go (hunk 2 above)
Comment (SteveHNH) [reply logger/logger.go]: yes we should. nice catch

**Pull request merged**